umm-spec-lib/src/cmr/umm_spec/models/umm_collection_models.clj

```diff
@@ -151,21 +151,17 @@
    ;; The title of the collection or service described by the metadata.
    EntryTitle
 
-   ;; This element describes the production status of the data set. There are multiple choices for Data
+   ;; This element describes the production status of the data set. There are five choices for Data
```

Comment:
if the number changes in the future no one will know this comment needs updated as well, I think multiple is preferable